Cabal/Cabal.cabal

```diff
@@ -188,6 +188,8 @@ library
       Win32 >= 2.2 && < 2.4
 
   ghc-options: -Wall -fno-ignore-asserts -fwarn-tabs
+  if impl(ghc >= 8.0)
```

Comment:
...sadly, `cabal-install` isn't yet GHC8-ready (mostly due to its build-deps) :-/